maestro-studio/web/src/commandExample.ts

```diff
@@ -0,0 +1,125 @@
+import { UIElement } from './models';
+import YAML from 'yaml';
+
+export type CommandExample = {
+  status: 'available' | 'unavailable'
+  title: string
+  content: string
+  documentation: string
+}
+
+type Selector = {
+  title: string
+  status: 'available'
+  definition: any
+  documentation?: string
+} | {
+  title: string
+  status: 'unavailable'
+  message: string
+  documentation?: string
+}
+
+const toPercent = (n: number, total: number) => `${Math.round((100 * n / total))}%`
+
+const getCoordinatesSelector = (deviceWidth: number, deviceHeight: number, uiElement: UIElement): Selector => {
+  const bounds = uiElement.bounds || { x: 0, y: 0, width: 0, height: 0 }
+  const cx = toPercent(bounds.x + bounds.width / 2, deviceWidth)
+  const cy = toPercent(bounds.y + bounds.height / 2, deviceHeight)
+  return {
+    status: 'available',
+    title: 'Coordinates',
+    definition: { point: `${cx},${cy}` }
+  }
+}
+
+const getSelectors = (uiElement: UIElement): Selector[] => {
+  const selectors: Selector[] = []
+  if (uiElement.resourceId) {
+    if (typeof uiElement.resourceIdIndex === 'number') {
+      selectors.push({
+        title: 'Resource Id',
+        status: 'available',
+        definition: {
+          id: uiElement.resourceId,
+          index: uiElement.resourceIdIndex,
+        }
+      })
+    } else {
+      selectors.push({
+        title: 'Resource Id',
+        status: 'available',
+        definition: {
+          id: uiElement.resourceId,
+        }
+      })
+    }
+  } else {
+    selectors.push({
+      title: 'Resource Id',
+      status: 'unavailable',
+      message: 'This element has no resource id associated with it. Type ‘\u2318 D’ to view platform-specific documentation on how to assign resource ids to ui elements.',
```

Comment:
Nit: can we implement an inline code snippet component instead of using backticks for `\u2318 D`?